javatests/dagger/functional/cycle/DoubleCheckCycleTest.java

```diff
@@ -157,6 +157,7 @@ public void testFailingReentrant() {
   }
 
   @Test(timeout = 5000)
+
```

Comment:
maybe it's new line unnecessary.
```suggestion
```

Reply:
this is actually a bug in MOE: we have a special test annotation that isn't opensourced, and it looks like MOE stripped the annotation but not the empty line.

Reply:
If we moved the internal annotation above the `@Test` annotation that would probably fix this since MOE will collapse multiple consecutive blank lines.